html-forms/html-forms/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>html-forms</title>
+  </head>
+  <body>
+    <h1><b>Coding Form!</b></h1>
+    <form>
+      <div>
+        <label>Email<br  /><input type="text" name="email"></label>
+      </div>
+      <div>
+        Why is coding cool? <br  /><select name="reasons">
```

Comment:
make sure to use a label for this input, with the proper `for` attritbute and matching `id` attribute on the input